### PR TITLE
Asynchronous requests

### PR DESCRIPTION
**Pull request by itsamirhn**
Just for cleaner code & faster usage, I used async & await feature.
As I added a timer, It seems this changes make it about 0.2 seconds faster.

### CODE REVIEW

**Review by iw4p — COMMENTED** [review]
removing version of httpx in req.txt

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 beautifulsoup4==4.11.1
 requests==2.27.1
 colorama==0.4.4
-urllib3==1.26.9
\ No newline at end of file
+urllib3==1.26.9
+httpx~=0.23.0
```
Comment (iw4p) [inline requirements.txt]: extra ~

File: proxyScraper.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import argparse
+import asyncio
 import re
-import threading
+import httpx
```
Comment (iw4p) [inline proxyScraper.py]: isort - import order

File: proxyScraper.py [hunk 3]
```diff
@@ -165,4 +164,4 @@ def scrape_scraper(scraper):
     )
     args = parser.parse_args()
 
-    scrape(method=args.proxy, output=args.output, verbose=args.verbose)
+    asyncio.run(scrape(args.proxy, args.output, args.verbose))
```
Comment (iw4p) [inline proxyScraper.py:167]: module 'asyncio' has no attribute 'run'

File: proxyScraper.py (hunk 2 above)
Comment (iw4p) [reply proxyScraper.py]: fixed.

File: proxyScraper.py (hunk 3 above)
Comment (iw4p) [reply proxyScraper.py:167]: Because the asyncio.run() function was added in Python 3.7

File: requirements.txt (hunk 1 above)
Comment (iw4p) [reply requirements.txt]: removed - fixed

**Review by iw4p — COMMENTED** [review]
Thank you so much! but there are some bugs.

**Comment by itsamirhn** [comment]
It seems you fixed the issue of the Tests with python >= 3.7; Do you mean the issue of python 3.6 or you find another bug?

**Comment by itsamirhn** [comment]
Finally the checks have passed :)

**Comment by iw4p** [comment]
Yay! Thank you so much. I learned a lot from your codes!

**Pull request merged**